### PR TITLE
Refactor docs

### PR DESCRIPTION
**Pull request by r2d4**
ref #1364

### CODE REVIEW

File: docs/contributors/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Contributing guidelines
```
Comment (dlorenc) [inline docs/contributors/CONTRIBUTING.md]: Can we keep this one at the top-level? I think CONTRIBUTING.md at the top-level is used by other systems, simlar to LICENSE, etc.

Comment (r2d4) [inline docs/contributors/CONTRIBUTING.md]: yeah makes sense

Comment (r2d4) [inline docs/contributors/CONTRIBUTING.md]: done

**Comment by r2d4** [comment]
@dlorenc @aaron-prindle PTAL

Summary of changes

* README.md split up into specific tutorials in `./docs`
* `brew` as preferred installation on macOS
* `installation guide on README
* minikube iso changelog merged with minikube changelog
* developer guides moved to `./docs/contributors`
* no relative links on main `README`, so that we can port this over to kubernetes.io

**Pull request merged**